src/jsonrpc/server/utils.rs

```diff
@@ -8,37 +8,26 @@ use crate::{
 };
 
 macro_rules! expects_fd {
-    ($fd_opt:expr, $ret:ident, $default:expr, true) => {
```

Comment:
It looks like ```$ret``` has been made unnecessary as well.

Reply:
Okay, resolved!